### PR TITLE
[UCHAT-4566] Email Channel Members on 1 click. 

### PR DESCRIPTION
**Pull request by nykashish**
Web App Implementation of send email to all members on 1-click.
Redux and Server changes linked included at bottom.

#### Summary

A new Email Member Button will be rendered depending on the config EnableEmailSendInChannel.
Once clicked, it will fetch all the members of the channel, and open it in email client.

Screenshot 2019-10-15 at 2 42 35 PM

#### Ticket Link
https://jira.uberinternal.com/browse/UCHAT-4566

#### Related Pull Requests

- Has server changes Yes. https://github.com/uber-uchat/mattermost-redux/commit/01d2941f0017caf07bd6e02d83d3d825c51a6c7d
- Has redux changes Yes. https://github.com/uber-uchat/mattermost-server/commit/7d0131875f6dcd9d3b3517249b68571bf8a16a5d
- Has mobile changes (please link here)

### CODE REVIEW

File: components/email_members/email_members.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        }).isRequired,
+    };
+
+    handleEmailChannelMembers = () => {
+        this.props.actions.getChannelMembersEmails(this.props.channelId).then((data) => {
+            window.location.href = 'mailto:' + data.data;
+        });
+    };
+
+    render() {
```
Comment (amitmall) [inline components/email_members/email_members.jsx:24]: check EnableEmailSendInChannel and do early return.

File: components/email_members/email_members.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                >
+                    <OverlayTrigger
+                        trigger={['hover', 'focus']}
+                        placement='bottom'
+                        overlay={emailMembersTooltip}
+                    >
+                        <div
+                            id='member_email'
+                            onClick={this.handleEmailChannelMembers}
+                        >{icon}
```
Comment (amitmall) [inline components/email_members/email_members.jsx]: icon on new line.

**Review by amitmall — CHANGES_REQUESTED** [review]

File: components/email_members/email_members.jsx (hunk 2 above)
Comment (nykashish) [reply components/email_members/email_members.jsx]: Done the changes. Github link
https://github.com/uber-uchat/mattermost-webapp/pull/80/commits/1ebd689756052660d21a4623394f238fbec04e66

**Review by amitmall — APPROVED** [review]
Looks good to me.

**Pull request merged**